lib/iex/lib/iex/introspection.ex

```diff
@@ -115,13 +126,35 @@ defmodule IEx.Introspection do
         end
         :ok
       else
-        :not_found
+        if has_callback?(mod, fun, arity) do
+          :behaviour_found
+        else
+          :not_found
+        end
       end
     else
       :no_docs
     end
   end
 
+  defp has_callback?(mod, fun) do
+    mod
+    |> Code.get_docs(:callback_docs)
+    |> find_callback_doc(fun)
+  end
+  defp has_callback?(mod, fun, arity) do
```

Comment:
Let's leave a space between these two clauses. Same for other things like `find_callback_doc/2,3` below.